app/secrets.py

```diff
@@ -20,8 +24,8 @@ def validate_required_secrets(secrets, additional_required_secrets=None):
 
 
 class SecretStore:
-    def __init__(self, secrets):
+    def __init__(self, secrets: dict) -> None:
```

Comment:
Can `secrets` be more specific here like the `Mapping[str, str]` above?

Reply:
I did try this initially but it caused errors because str has no get function on line 28

Reply:
This should be match the type hint from my previous comment above. It's a mapping of mappings instead of mapping of strings.